app/presenters/bid_status_presenter/future_user_is_admin.rb

```diff
@@ -0,0 +1,31 @@
+class BidStatusPresenter::FutureUserIsAdmin < BidStatusPresenter::Base
+  include Rails.application.routes.url_helpers
+  include ActionView::Helpers::UrlHelper
+
+  attr_reader :auction
+
+  def initialize(auction:)
+    @auction = auction
+  end
+
+  def body
+    "This auction is visible to the public but is not currently accepting bids.
+    It will open on #{start_date}. If you need to take it down for whatever
+    reason, press the unpublish button below. #{link}"
+  end
+
+  private
+
+  def link
+    link_to(
+      "Un-publish",
```

Comment:
I always wonder if we should put link_to or other `view` things in Presenters or try to put them up in ViewModels. It's a hard call though, since the latter often involves some sort of weird checking or stuff that belongs within the Presenter